### DISCUSSION TITLE
[DataFilterExtension] About the number of filters

### DISCUSSION THREAD

**Discussion by zogs** [visgl/deck.gl#8712] [Q&A]
Hello !
Is there a reason for the limit of 4 filters maximum ?
What would be needed to increase this capacity ?

Thank you kindly

**Comment by Pessimistress** [comment]
The reason is that a shader attribute can be a maximum of vec4. There is also a limit of 16 attributes for a WebGL shader, so depending on the layer you are using, you may not have many more to spare.
